clientUtils/Util.ts

```diff
@@ -921,6 +921,17 @@ export const differenceOfSets = <T>(sets: Set<T>[]): Set<T> => {
     return diff
 }
 
+/** Tests whether the first argument is a strict subset of the second. The arguments do not have
+    to be sets yet, they can be any iterable. Sets will be created by the function internally */
+export function isStrictSubsetOf<T>(
+    subsetIter: Iterable<T>,
+    supersetIter: Iterable<T>
+): boolean {
```

Comment:
Thank you very much for catching this, Marcel! I did this in passing and it's been a while since I checked my maths terminology. I'll rename it to isSubset of.